plugins/modules/eos_vrf_global.py

```diff
@@ -0,0 +1,461 @@
+- name: Delete the provided configuration
+  arista.eos.eos_vrf_global:
+    config:
+    state: deleted
+
+# Task Output:
+# ------------
+#
+# before:
+# - name: VRF4
+# - name: VRF6
+#   description: VRF6 Description
+#   rd: "9:8"
+# - name: VRF7
+
+# commands:
+# - vrf instance VRF4
+# - vrf instance VRF6
+# - no description VRF6 Description
+# - no rd 9:8
+# - vrf instance VRF7
+#
+# after:
+# - name: VRF4
+# - name: VRF6
+# - name: VRF7
+#
+# After state:
+# ------------
+#
+# test#show running-config | section ^vrf
+# vrf instance VRF4
+# vrf instance VRF6
+# vrf instance VRF7
+
+# Using purged
+#
+# Before state:
+# -------------
+#
+# test#show running-config | section ^vrf
+# vrf instance VRF4
+# vrf instance VRF6
+# vrf instance VRF7
+
+- name: Purge all the configuration from the device
+  arista.eos.eos_vrf_global:
+    state: purged
+
+# Task Output:
+# ------------
+#
+# before:
+# - name: VRF4
+# - name: VRF6
+# - name: VRF7
+#
+# commands:
+# - no vrf instance VRF4
+# - no vrf instance VRF6
+# - no vrf instance VRF7
+#
+# after: []
+#
+# After state:
+# -------------
+# test#show running-config | section ^vrf
+# -
+
+# Using rendered
+#
+- name: Render provided configuration with device configuration
+  arista.eos.eos_vrf_global:
+    config:
+      - name: VRF4
+        description: VRF4 Description
+        rd: "3:4"
+    state: rendered
+
+# Task Output:
+# ------------
+#
+# rendered:
+# - vrf instance VRF4
+# - description VRF4 Description
+# - rd 3:4
+
+# Using gathered
+#
+# Before state:
+# -------------
+#
+# test#show running-config | section ^vrf
+# vrf instance VRF4
+#  description "VRF4 Description"
+#  rd "3:4"
+
+- name: Gather existing running configuration
+  arista.eos.eos_vrf_global:
+    state: gathered
+
+# Task Output:
+# ------------
+#
+# gathered:
+# - name: VRF4
+#   description: VRF4 Description
+#   rd: "3:4"
+
+# Using parsed
+#
+# File: parsed.cfg
+# ----------------
+#
+# vrf instance test
+#  description "This is test VRF"
+#  rd "testing"
+#  !
+# !
+# vrf my_vrf
+#  description "this is sample vrf for feature testing"
+#  rd "2:3"
+#  !
+# !
+
+- name: Parse the provided configuration
+  arista.eos.eos_vrf_global:
+    running_config: "{{ lookup('file', 'parsed.cfg') }}"
+    state: parsed
+
+# Task Output:
+# ------------
+#
+# parsed:
+# - description: This is test VRF
+#   name: test
+#   rd: testing
+# - description: this is sample vrf for feature testing
+#   name: my_vrf
+#   rd: '2:3'
+"""
+
+RETURN = """
+before:
+  description: The configuration prior to the module execution.
+  returned: when I(state) is C(merged), C(replaced), C(overridden), C(deleted) or C(purged)
+  type: dict
+  sample: >
+    This output will always be in the same format as the
+    module argspec.
+after:
+  description: The resulting configuration after module execution.
+  returned: when changed
+  type: dict
+  sample: >
+    This output will always be in the same format as the
+    module argspec.
+commands:
```

Comment:
can we have actual example for this ?

Reply:
i completely ignored this section, updating it!